packages/manager/modules/pci/src/projects/project/instances/instance/edit/edit.routing.js

```diff
@@ -60,7 +60,7 @@ export default /* @ngInject */ ($stateProvider) => {
       },
       imageInformation: /* @ngInject */ ($http, projectId, instance) => {
         return $http
-          .get(`/cloud/project/${projectId}/image/${instance.image.id}`)
+          .get(`/cloud/project/${projectId}/image/${instance.image?.id}`)
```

Comment:
So if image is undifined you call .get(`/cloud/project/${projectId}/image/undefined`) i think this case shoudn't exist.

Reply:
Sometimes instance dont have images for few instances, I will add a if condition to resolve this issue.